ReadMe.md

```diff
@@ -366,7 +366,9 @@ general:
   # The format for this env variable is "src_db1:target_db1,src_db2:target_db2". For YAML please continue using map syntax
   restore_database_mapping: {}   
   retries_on_failure: 3          # RETRIES_ON_FAILURE, how many times to retry after a failure during upload or download
-  retries_pause: 30s             # RETRIES_PAUSE, duration time to pause after each download or upload failure 
+  retries_pause: 30s             # RETRIES_PAUSE, duration time to pause after each download or upload failure
+
+  sharded_operation: false       # SHARDED_OPERATION, backups to replicas will save the schema but will only save one replica copy of the data with tables sharded among replicas.
```

Comment:
Moreover, will different tables back up data from different replicas depends on `db.table` key?
Am I right?
Then from my perspective it would less data consistency between tables in backup for some corner cases.

Reply:
This parameter was changed from `sharded_operation` to `sharded_operation_mode`. The new parameter is a string and can take `none`, `table`, `database`, or `first-replica`. `none` applies no sharding (back up all tables on all replicas), `table` shards by individual tables, `database` shards by database, and `first-replica` shards such that tables will be backed up on the lexicographically first active replica.